cryptoshred-core/src/test/java/eu/prismacapacity/cryptoshred/core/CryptoObjectMapperTest.java

```diff
@@ -4,24 +4,22 @@
 
 import java.util.UUID;
 
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
 
 import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
-import eu.prismacapacity.cryptoshred.core.keys.CryptoKeySize;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
+import lombok.*;
 
 public class CryptoObjectMapperTest {
```

Comment:
There is no more `CryptoObjectMapper`, so maybe this class should be renamed.

Reply:
oh, i missed that one, will do. Thanks for catching!